---
realman_interface:
  joint_names:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
    - joint7
  publish_rate: 30.0
  follow: false
  expand: 0.0

